internal/api/v2/types_gen.go

```diff
@@ -91,11 +96,18 @@ type TxResponse struct {
 }
 
 type UrlQuery struct {
-	Url   string `json:"url,omitempty" form:"url" query:"url" validate:"required,acc-url"`
+	Url string `json:"url,omitempty" form:"url" query:"url" validate:"required,acc-url"`
+}
+
+type QueryPagination struct {
 	Start uint64 `json:"start,omitempty" form:"start" query:"start"`
 	Count uint64 `json:"count,omitempty" form:"count" query:"count"`
```

Comment:
Since this is a separate interface now, maybe the validation for count can be set to required.